<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AliasImportDialog</class>
 <widget class="QDialog" name="AliasImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alias Import Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="aliasimport">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="importoffers">
      <attribute name="title">
       <string>Import &amp;Offers</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>311</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="importOfferDisclaimer">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Category:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="offerCategory"/>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="offerSafeSearch">
          <property name="text">
           <string>Safe Search</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="importOffers">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Import Offers</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="importcerts">
      <attribute name="title">
       <string>Import &amp;Certificates</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>311</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="importCertDisclaimer">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Category:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="certCategory"/>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="certSafeSearch">
          <property name="text">
           <string>Safe Search</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="importCerts">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Import Certificates</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="importescrows">
      <attribute name="title">
       <string>Import &amp;Escrows</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>311</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="importEscrowDisclaimer">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="importEscrows">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Import Escrows</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../syscoin.qrc"/>
 </resources>
 <connections/>
</ui>
